app/client/components/helpCentre/helpCentre.tsx

```diff
@@ -71,6 +72,8 @@ const HelpCentre = (_: HelpCentreProps) => (
             <HelpTopicBox key={topic.id} topic={topic} />
           ))}
         </div>
+        <h2 css={contactUsTitleStyle}>FAQ</h2>
```

Comment:
`contactUsTitleStyle` is now being used for more than just the Contact Us title. Can we rename it so it has a more meaningful name?

Reply:
What title would you suggest?

Reply:
Something like 'subtitleStyles' seems appropriate. What do you think?